VD_Transmit/VD_Transmit.ino

```diff
@@ -0,0 +1,177 @@
+  //Serial.println("LSM6DS and LIS3MDL Found!");
+  
+  /*******************************************************************************/
+  /* Configure accelerometer range */
+  lsm6ds.setAccelRange(LSM6DS_ACCEL_RANGE_2_G);
+
+  /* Configure accelerometer data rate */
+  lsm6ds.setAccelDataRate(LSM6DS_RATE_208_HZ);
+
+  /* Configure gyro range */
+  lsm6ds.setGyroRange(LSM6DS_GYRO_RANGE_250_DPS);
+
+  /* Configure gyro data rate */
+  lsm6ds.setGyroDataRate(LSM6DS_RATE_208_HZ);
+
+  /* Configure magnetometer data rate */
+  lis3mdl.setDataRate(LIS3MDL_DATARATE_155_HZ);
+  // You can check the datarate by looking at the frequency of the DRDY pin
+
+  /* Configure magnetometer data range */
+  lis3mdl.setRange(LIS3MDL_RANGE_4_GAUSS);
+
+  /* Configure magnetometer performance mode */
+  lis3mdl.setPerformanceMode(LIS3MDL_HIGHMODE);
+
+  /* Configure magnetometer operation mode */
+  lis3mdl.setOperationMode(LIS3MDL_CONTINUOUSMODE);
+
+  lis3mdl.setIntThreshold(500);
+  lis3mdl.configInterrupt(false, false, true, // enable z axis
+                          true, // polarity
+                          false, // don't latch
+                          true); // enabled!
+                          
+  /*******************************************************************************/
+}
+
+void loop()
+{
+  /**********************************************************************************************/
+  /*************************************[BEGIN IMU LOOP]*****************************************/
+  /**********************************************************************************************/
+  sensors_event_t accel, gyro, mag, temp;
+
+  /* Get new normalized sensor events */
+  lsm6ds.getEvent(&accel, &gyro, &temp);
+  lis3mdl.getEvent(&mag);
+
+  /**********************************************************************************************/
+  /********************************[BEGIN CAN-BUS PROTOCOL LOOP]**********************************/
+  /**********************************************************************************************/
+  if(CAN_Send_Delay.justFinished()) // Checking the delay on each loops
+  {
+    // Delay has expired so set it to run again. 
+    // This library does not allow drift and logs the actual expiration time. 
+    CAN_Send_Delay.repeat(); 
+
+    /**********************************************************************************************/
+//    /* DEBUG PRINT LINES */
+//    //Serial.println("Sending the following values through CAN");
+//    // Acceleration data
+//    Serial.print(accel.acceleration.x);Serial.print(" ");
+//    Serial.print(accel.acceleration.y);Serial.print(" ");
+//    Serial.print(accel.acceleration.z);Serial.print(" ");
+//    // Gyroscope data
+//    Serial.print(gyro.gyro.x);Serial.print(" ");
+//    Serial.print(gyro.gyro.y);Serial.print(" ");
+//    Serial.print(gyro.gyro.z);Serial.print(" ");
+//    // Magnetometer data
+//    Serial.print(mag.magnetic.x);Serial.print(" ");
```

Comment:
I'd like to do a trial with some logging libraries so you're not stuck commenting stuff out like this. We can just have a enable / disable at the top.

Reply:
I'd like that too.